components/telegram_bot_api/actions/create-chat-invite-link/create-chat-invite-link.mjs

```diff
@@ -0,0 +1,52 @@
+import telegramBotApi from "../../telegram_bot_api.app.mjs";
+
+export default {
+  key: "telegram_bot_api-create-chat-invite-link",
+  name: "Create Chat Invite Link",
+  description: "Use this method to create an additional invite link for a chat. The bot must be an administrator in the chat for this to work and must have the appropriate administrator rights. The link can be revoked using the method revokeChatInviteLink. Returns the new invite link as ChatInviteLink object. Note: Each administrator in a chat generates their own invite links. Bots can't use invite links generated by other administrators. If you want your bot to work with invite links, it will need to generate its own link using exportChatInviteLink or by calling the getChat method. If your bot needs to generate a new primary invite link replacing its previous one, use exportChatInviteLink again. [See the docs](https://core.telegram.org/bots/api#createchatinvitelink) for more information",
```

Comment:
This description is going to be visible in the Pipedream dashboard for the user to read.

Try to keep this succinct to 1 sentence if you can. Here's an example: https://github.com/PipedreamHQ/pipedream/blob/750ff1d6cde94838d54a3bb48097d5d751b3e60c/components/telegram_bot_api/actions/edit-media-message/edit-media-message.mjs#L10